lib/backports/2.7.0/comparable/clamp.rb

```diff
@@ -0,0 +1,22 @@
+require 'backports/2.4.0/comparable/clamp' unless Comparable.method_defined? :clamp
+
+if Comparable.instance_method(:clamp).parameters.map(&:first) == [:req, :req]
+  require 'backports/tools/alias_method_chain'
+
+  module Comparable
+    def clamp_with_range(range_or_min, max = nil)
```

Comment:
Best use `Backports::Undefined`.

Also, there's clearly a missing spec for `5.clamp(1, nil)`.